ginkgo.Jenkinsfile

```diff
@@ -54,66 +53,173 @@ pipeline {
                }
             }
         }
-        stage('Boot VMs'){
+        stage ("copy-code-and-boot-vms"){
             options {
                 timeout(time: 30, unit: 'MINUTES')
             }
+
             environment {
-                TESTDIR="${WORKSPACE}/${PROJ_PATH}/test"
+                FAILFAST=setIfLabel("ci/fail-fast", "true", "false")
+                CONTAINER_RUNTIME=setIfLabel("area/containerd", "containerd", "docker")
             }
-            steps {
-                sh 'cd ${TESTDIR}; NETNEXT=true K8S_VERSION=1.10 vagrant up --no-provision'
-                sh 'cd ${TESTDIR}; K8S_VERSION=1.14 vagrant up --no-provision'
+            parallel {
+                stage('boot-vms-runtime') {
+                    environment {
+                        GOPATH="${WORKSPACE}/runtime-gopath"
+                        TESTDIR="${GOPATH}/${PROJ_PATH}/test"
+                    }
+                    steps {
+                        sh 'mkdir -p ${GOPATH}/src/github.com/cilium'
+                        sh 'cp -a ${WORKSPACE}/${PROJ_PATH} ${GOPATH}/${PROJ_PATH}'
+                        sh 'cd ${TESTDIR}; vagrant up runtime --provision'
+                    }
+                    post {
+                        failure {
+                            script {
+                                if  (currentBuild.description == '') {
```

Comment:
Since we have the string already, perhaps we can just append to it? Maybe also mark this as a generated message (but that might make it too long).
Separately, have we tested how these look in the jenkins UI? do they scroll off and you only see `runtime vm`? (which is ok, I guess)

Reply:
I am not appending here, because other parallel builds will be terminated if this particular build fails and they can either be in failed or aborted state, but we want to get the message from the parallel step that failed firsts.

Reply:
Also, seems like the description bit doesn't work yet.

Reply:
ah, I see. That hadn't occurred to me. Can we add comments to this file? a comment explaining why this is the way it is will be helpful the next time we need to change things.